tests/e2e/page-layouts-modal.spec.js

```diff
@@ -0,0 +1,13 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('checks if the welcome modal on the editor is present and closed when the button is clicked', async ({page, context}) => {
+  test.setTimeout(240 * 1000);
```

Comment:
Feel free to remove this `timeout` if you think it's not really necessary through this `spec`